### PR TITLE
build: add ft when PR is approved

### PR DESCRIPTION
**Pull request by kairoaraujo**
Add the FT for when a PR is approved.

It is also used during the API release.

# Description

Fixes #(issue)

# Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)

# Additional requirements
- [x] Tests have been added for the bug fix or new feature
- [ ] Docs have been added for the bug fix or new feature

# Code of Conduct
By submitting this PR, you agree to follow our [Code of Conduct](CODE_OF_CONDUCT.rst).

- [x] I agree to follow this project's Code of Conduct

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -90,9 +90,6 @@ services:
       - DATA_DIR=./data
       - RSTUF_BROKER_SERVER="amqp://guest:guest@rabbitmq:5672"
       - RSTUF_REDIS_SERVER="redis://redis"
-      - RSTUF_AUTH=true
-      - SECRETS_RSTUF_TOKEN_KEY="secret"
-      - SECRETS_RSTUF_ADMIN_PASSWORD="secret"
```
Comment (MVrachev) [inline docker-compose.yml:95]: Please, let's make such changes in a specific commit with a commit message with arguments as to why we do that, so we can reference this later.

**Review by MVrachev — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (MVrachev) [reply docker-compose.yml:95]: That's for future prs.

**Comment by kairoaraujo** [comment]
@MVrachev 

All tests are ok.
https://github.com/repository-service-tuf/repository-service-tuf-api/actions/runs/6348766054/job/17245960598?pr=458

LGTM to merge.

File: docker-compose.yml (hunk 1 above)
Comment (MVrachev) [reply docker-compose.yml:95]: My point was to do that in another commit, not another pr.

**Review by MVrachev — APPROVED** [review]

**Comment by MVrachev** [comment]
> @MVrachev
> 
> All tests are ok. https://github.com/repository-service-tuf/repository-service-tuf-api/actions/runs/6348766054/job/17245960598?pr=458
> 
> LGTM to merge.

Force-squashed as tests already passed.

**Pull request merged**